docs/apis-hook.md

```diff
@@ -4,4 +4,149 @@ sidebar_position: 4
 
 # APIs hook
 
-This page will discuss - How to interact with the ReactPlay data using custom hook.
+In React Play, we work with APIs by using our custom-made react hooks. They are as follows:
+
```

Comment:
Are we following an alphabetical order here?

Reply:
Yes, the hooks are listed in alphabetical order. Let me know if it has to be changed to a particular order.